script/.ipynb_checkpoints/download_MNIST-checkpoint.py

```diff
@@ -0,0 +1,76 @@
+__author__ = "Xinqiang Ding <xqding@umich.edu>"
+
+import numpy as np
+import urllib3
+import gzip
+import subprocess
+import pickle
+
+# download train labels
+print("Downloading train-labels-idx1-ubyte ......")
+http = urllib3.PoolManager()
+r = http.request('GET', 'http://yann.lecun.com/exdb/mnist/train-labels-idx1-ubyte.gz')
```

Comment:
Noticing the domain name and presuming this is sourced from Yann LeCun's work somehow, would it make sense to make a citation about where this data came from for the repo? Additionally, I recommend being aware of any licensing constraints surrounding the data.

Reply:
added citation